tests/test_abc.py

```
@@ -804,3 +805,29 @@ async def test_unexpected_status_code(self):
         with pytest.raises(GraphQLException) as exc:
             await gh.graphql("does not matter")
         assert exc.value.response == response_data
+
+    @pytest.mark.asyncio
+    async def test_response_content_type_parsing_gh121(self):
+        gh, response_data = self.gh_and_response("success-200.json")
+        # Test that a json content type still works if formatted without spaces
+        gh.response_headers["content-type"] = "application/json;charset=utf-8"
+        # Should not fail
+        resp = await gh.graphql("does not matter")
+        assert resp is not None
+
+    @pytest.mark.asyncio
+    async def test_unknown_response_content_type_gh121(self):
+        gh, response_data = self.gh_and_response("success-200.json")
+        # Test that a json content type still works if formatted without spaces
+        gh.response_headers["content-type"] = "application/gidget;charset=utf-8"
+        with pytest.raises(GraphQLResponseTypeError) as exc:
+            await gh.graphql("does not matter")
+        assert exc is not None
+
+    @pytest.mark.asyncio
+    async def test_no_response_content_type_gh121(self):
+        gh, response_data = self.gh_and_response("success-200.json")
+        # Test that a json content type still works if formatted without spaces
+        gh.response_headers["content-type"] = ""
+        resp = await gh.graphql("does not matter")
```

This should raise an exception.